### PR TITLE
release-v1.12.0

### PR DESCRIPTION
**Pull request by JohannesDoberer**
#### :rocket: Added
* [#1841](https://github.com/SAP/luigi/pull/1841) Bookmarkable modals ([@maxmarkus](https://github.com/maxmarkus))
* [#1935](https://github.com/SAP/luigi/pull/1935) Add warning if both virtualTree and children nodes are defined  ([@ndricimrr](https://github.com/ndricimrr))
* [#1894](https://github.com/SAP/luigi/pull/1894) Add more flexibility to control handshake ([@legteodav](https://github.com/legteodav))

#### :bug: Fixed
* [#1942](https://github.com/SAP/luigi/pull/1942) Fix 'disabled loadingIndicator' bug when opening in modal ([@ndricimrr](https://github.com/ndricimrr))
* [#1916](https://github.com/SAP/luigi/pull/1916) Shell header improvements ([@JohannesDoberer](https://github.com/JohannesDoberer))
* [#1912](https://github.com/SAP/luigi/pull/1912) Change color of active link ([@JohannesDoberer](https://github.com/JohannesDoberer))

### CODE REVIEW

**Review by stanleychh — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,26 @@ The lerna-changelog tool detects changes based on PR labels and maps them to sec
...
+## [v1.12.0] (2021-03-24)
+
+#### :rocket: Added
+* [#1841](https://github.com/SAP/luigi/pull/1841) Bookmarkable modals ([@maxmarkus](https://github.com/maxmarkus))
+* [#1935](https://github.com/SAP/luigi/pull/1935) Add warning if both virtualTree and children nodes are defined  ([@ndricimrr](https://github.com/ndricimrr))
+* [#1894](https://github.com/SAP/luigi/pull/1894) Add more flexibility to control handshake ([@legteodav](https://github.com/legteodav))
+* [#1897](https://github.com/SAP/luigi/pull/1897) Update fiddle to v1.11.0 ([@JohannesDoberer](https://github.com/JohannesDoberer))
+
+#### :bug: Fixed
+* [#1945](https://github.com/SAP/luigi/pull/1945) Quick Fix Console Warning ([@ndricimrr](https://github.com/ndricimrr))
```
Comment (ndricimrr) [inline CHANGELOG.md:37]:
```suggestion
```
I'd remove this, as its follow-up of 1935 in Added section

Comment (ndricimrr) [reply CHANGELOG.md:37]: Aside from that, looks good 👍

Comment (JohannesDoberer) [reply CHANGELOG.md]: thanks.. updated in the description.. but didn't commit it :)

**Review by ndricimrr — APPROVED** [review]
👍

**Pull request merged**